src/Chapter2.hs

```diff
@@ -740,8 +750,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains x = filter (elem x)
```

Comment:
Nice work on polymorphic type signatures and eta-reduced implementation! :trophy: